package.json

```diff
@@ -2,14 +2,66 @@
   "name": "react-slider",
   "version": "0.11.2",
   "description": "Slider component for React",
-  "main": "react-slider.js",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "sideEffects": false,
+  "files": [
+    "es",
+    "lib",
+    "src",
+    "docs",
+    "styleguide.config.js"
```

Comment:
Don't need src, docs, or styleguide.config.js since we aren't linking this ever.